contributions/api/contribution_jsons.py

```diff
@@ -0,0 +1,33 @@
+#  Copyright 2020 Board of Trustees of the University of Illinois.
```

Comment:
Please change this to 2022. When a new file is added it needs to use the year in which that file was created. I think the discussion in Slack was about updating the year in existing files, which may not be needed.

Reply:
this file has been removed =)

Reply:
Thanks.